RecoTracker/MkFitCore/src/CandCloner.cc

```diff
@@ -191,7 +191,7 @@ namespace mkfit {
           if (n_pushed >= mp_iteration_params->maxCandsPerSeed)
             break;
 
-          // set the overlap if we have a true hit and pT > pTCutOverlap
+          // set the overlap if we have a true hit and pT > pTCutOverlap (=0 by default)
```

Comment:
```suggestion
          // set the overlap if we have a true hit and pT > pTCutOverlap
```
is it really useful to have the default value mentioned explicitly here? It would usually refer to a hardcoded default.
There is no formal mechanism for a default value, if the value is configurable and is always taken from config.

Reply:
Done.